guides/Makefile

```diff
@@ -1,37 +1,51 @@
 SHELL := /bin/bash
-ALL_DIRS = $(shell ls -d doc-*)
-GUIDES-HTML = $(ALL_DIRS)
+ALL_DIRS = $(wildcard doc-*)
+ALL_GUIDES = $(patsubst doc-%,%,$(ALL_DIRS))
+BUILDS=foreman-el foreman-deb katello
+BUILD_DIR := build
+COMMON_DIR := common
+DEPENDENCIES := $(shell find $(COMMON_DIR) -type f -name \*.adoc)
+DATE_MDY := $(shell LC_ALL=C date "+%b %d %Y")
+DATE_MY := $(shell LC_ALL=C date "+%b %Y")
 
-.PHONY: all html pdf linkchecker all-html all-pdf clean all-clean linkchecker-tryer $(GUIDES-HTML)
+HTML_INDEXES := $(BUILDS:%=index-%.html)
+PDFS := $(BUILDS:%=index-%.pdf)
+GUIDE_BUILDS := $(addprefix $(BUILD_DIR)/,$(ALL_GUIDES))
+
+GUIDES-HTML := $(foreach guide,$(GUIDE_BUILDS),$(addprefix $(guide)/,$(HTML_INDEXES)))
+GUIDES-PDF := $(foreach guide,$(GUIDE_BUILDS),$(addprefix $(guide)/,$(PDFS)))
+
+.PHONY: all html pdf linkchecker all-html all-pdf clean all-clean linkchecker-tryer
 
 all: html
 
-html: all-html
+html: $(GUIDES-HTML)
 
-pdf: all-pdf
+pdf: $(GUIDES-PDF)
 
-clean: all-clean
+clean:
+	rm -rf build
```

Comment:
```suggestion
	rm -rf $(BUILD_DIR)
```

Reply:
I swear I had that somewhere. So many branches.